src/electron/views/common/window-title/window-title.scss

```diff
@@ -14,7 +14,7 @@
     display: flex;
     align-items: center;
     -webkit-app-region: drag;
-    -webkit-user-select: none;
```

Comment:
This needs investigation.

Reply:
I think we need to add autoprefixer to webpack.config.js.

Reply:
Added in https://github.com/microsoft/accessibility-insights-web/pull/6278/commits/ccdbddf1d82061ec6ca24706c2cc9d9336f35dae, I confirmed by temporarily commit the drop files to see the diff.